### DISCUSSION TITLE
SSB

### DISCUSSION THREAD

**Discussion by ClintHennessey** [egzumer/uv-k5-firmware-custom#345] [Ideas]
WOULD BE NICE TO ADD SSB TO THE DEMODU BAND SELECTION AND W/N WITH WIDE + WIDE - AND NARROW+ NARROW - FOR SSB AND USB RX

**Comment by ErikS-web** [comment]
He Clint CAPS-lock defect ?

There is NO SSB; see Menu 13. 1 `Demodu` - demodulator mode, default is FM, AM/**USB can be used for listening only**

Can you be more specific with what you mean ?

**Comment by ClintHennessey** [comment]
Good evening
sorry about that was just wondering if could be done as seen on one of the other firmwares that person had SSB and had the choise of N N- W W-
here is the link to the bin file and the youtube
on what im trying to explain
(url)https://www.universirius.com/SirioArchive/Materiel_pr_site/Firmware-IJV/firmware_IJV_V2.3R5.bin
(url)https://www.youtube.com/watch?v=2U9zHlnLmmA)
Kind regard

**Comment by gartnl** [comment]
@ClintHennessey: Just about SSB: SSB means Single Side Band, and has two modes. Upper Side Band and Lower Side Band, or USB and LSB.
USB is the positive part of an AM signal without the carrier. LSB is the negative part but has the audio frequencies reversed, that is: a higher pitch has a lower rf frequency. USB is the most used modulation, by both HAM's and Pro's. 
IJV does not call it SSB but SB. But he means USB.

**Comment by brainzum** [comment]
@ClintHennessey
See this few videos for example and you will understand 

https://www.youtube.com/watch?v=EzuDTiOunnY
https://www.youtube.com/watch?v=Zepx-AoWfJU

No matter if you want to use SSB, DSB, CW, the egzumer 
firmware is completely deaf to operate well outside of FM. 
Don't need N W N+ filters and more modes or filters like 
LMS or SPEC all of them professionals ones because it is futile 
Another errors for solve forever also don't have sense .
Better change firm and forget empty phrases , it's the best thing you can 
do, migrate to a practical world and you will agree a lot

**Reply by HakanKoseoglu** [reply]
Not sure what antenna you use - with a suitable HF antenna I can hear SSB transmissions quite well on 28MHz and below.

**Comment by gartnl** [comment]
Just a mind-excersice:

Hypothetically you could receive LSB too. Tune the radio 3 kHz down from the actual frequency. The audio is than inverted as it is an LSB signal received on USB. Use the descrambler to invert it back to normal. ( scrambling is just old-fashioned frequency inversion ).

But the only use would be for those users that have an upconverter and listen to 80/40 meter. All other bands and pro's use USB.

**Comment by Hunterfnet** [comment]
Hypothetically and really this firmware needs more band-filters on main screen , not only BF , RF needed too,  and some unsolved  programming errors too for become a rival on conditions front others . If nothing changes no way for evolution 
I see God and angels "palmeros" have the truth, i know and i dont care bcz i'm happy with my family and friends xDD

**Reply by gartnl** [reply]
@Hunterfnet You keep complaining here. WHY? If this firmware does not suit you just leave. You are not contributing anything usefull.

**Reply by Hunterfnet** [reply]
gartnl 
i know your problem , why you keep here apporting nothing and making classes about a stup... LSB , USB ..etc ?. You read yourself ? You dont contributes nothing dude . Better get a family and stop trying going up on the sect .
First mature and become enginner is your main fault . Good year too ... improductible "palmero" better complains yourself .
